Modules/_ctypes/libffi_msvc/win32.c

```diff
@@ -2,21 +2,21 @@
    win32.S - Copyright (c) 1996, 1998, 2001, 2002  Red Hat, Inc.
 	     Copyright (c) 2001  John Beniton
 	     Copyright (c) 2002  Ranjit Mathew
-			
```

Comment:
I realize we have our own copy of libffi for Windows. Wouldn't changes to these files cause porting upstream changes more difficult? I suggest we not change any libffi files. See Modules/_ctypes/libffi_osx/README.ctypes.

Reply:
reverted.